networkx/generators/cographs.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+#    Copyright (C) 2004-2019 by
+#    Aric Hagberg <hagberg@lanl.gov>
+#    Dan Schult <dschult@colgate.edu>
+#    Pieter Swart <swart@lanl.gov>
+#    All rights reserved.
+#    BSD license.
+#
+# Authors: Efraim Rodrigues (efraimnaassom@gmail.com)
+r"""Generators for cographs
+
+A cograph is a graph containing no path on four vertices.
+Cographs or $P_4$-free graphs can be obtained from a single vertex
+by disjoint union and complementation operations.
+
+References
+----------
+.. [1] D.G. Corneil, H. Lerchs, L.Stewart Burlingham,
+       "Complement reducible graphs",
+    Discrete Applied Mathematics, Volume 3, Issue 3, 1981, Pages 163-174,
+    ISSN 0166-218X.
+"""
+import networkx as nx
+from networkx.utils import py_random_state
+
+__all__ = ['random_cograph']
+
+
+@py_random_state(1)
+def random_cograph(n, seed=None):
+    """Returns a random cograph with $2 ^ n$ nodes.
+
+    Parameters
+    ----------
+    n : int
+            The order of the cograph.
+    seed : integer, random_state, or None (default)
+            Indicator of random number generation state.
+            See :ref:`Randomness<randomness>`.
+
```

Comment:
Add a ```Returns``` block to the docstring
You might also want to use a paragraph before the ```parameters``` section to describe what a cograph is and how the random cograph is created.

Reply:
`Returns` block added. 

Copied information from the module docstring to the function docstring. Removed `Notes` since the information was inserted in the docstring as a paragraph.